content/blog/configure-isut-and-ams-for-vlcm-based-firmware-updates-on-gen-12-servers.md

```diff
@@ -0,0 +1,98 @@
+---
+title: How to Configure iSUT and AMS for vLCM-Based Firmware Updates on HPE
+  Gen12 Servers (High Security Mode)
+date: 2025-10-01T02:30:17.312Z
+author: Vijayakannan M
+authorimage: /img/Avatar1.svg
```

Comment:
If you have a picture of yourself, you can replace the default picture (avatar1.svg) with your author image, recommended resolution size: 192px x 192px.